backend/homework/4/Java-Fundamentals-4.iml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module version="4">
+  <component name="SonarLintModuleSettings">
```

Comment:
why is homework 4 change committed here? Anyways this file is supposed to be in gitignore